<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameResultDlg</class>
 <widget class="QDialog" name="GameResultDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>游戏结果</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_s">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>您的得分：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_t">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>用时:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_c">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>卡牌数:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cards">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_times">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_score">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_n">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>您的昵称：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushScore">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注：提交后，您的成绩将被上传到服务器，可查看您的当前排名.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GameResultDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GameResultDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
